GUI/src/app/pages/authorization-page/authorization-page.component.html

```diff
@@ -0,0 +1,15 @@
+  <div class="form-container">
+    <form class="authorization-form" [formGroup]="authorizationForm" (ngSubmit)="onSubmit()">
+      <h2 class="form-label">User Authorization</h2>
```

Comment:
Please, move all string constants to constant or enum in shared folder

Reply:
done